src/pages/iou/ReceiptSelector/index.native.js

```diff
@@ -37,15 +39,7 @@ const propTypes = {
         id: PropTypes.string,
         amount: PropTypes.number,
         currency: PropTypes.string,
-        participants: PropTypes.arrayOf(
-            PropTypes.shape({
-                accountID: PropTypes.number,
-                login: PropTypes.string,
-                isPolicyExpenseChat: PropTypes.bool,
-                isOwnPolicyExpenseChat: PropTypes.bool,
-                selected: PropTypes.bool,
-            }),
-        ),
+        participants: participantPropTypes,
```

Comment:
This caused a prop error.
1. participants should be at least set to PropTypes.arrayOf(participantPropTypes)
2. participantPropTypes does not include all the props that we need e.g. isPolicyExpenseChat, selected

Reply:
Oops, I will update the participantPropTypes to include the missing props